ws-client/src/manager.rs

```diff
@@ -53,6 +54,9 @@ pub struct RequestManager {
 	requests: FnvHashMap<RequestId, Kind>,
 	/// Reverse lookup, to find a request ID in constant time by `subscription ID` instead of looking through all requests.
 	subscriptions: HashMap<SubscriptionId, RequestId>,
+	/// Pending batch requests
+	// NOTE: BTreeSet to get sorted order of requestIDs.
+	batches: HashMap<BTreeSet<RequestId>, (Vec<RequestId>, PendingBatchOneshot)>,
```

Comment:
`FnvHashMap<Vec<RequestId>, (Vec<RequestId>, PendingBatchOneshot)>,` and sort the Ids before accessing the map.